pkg/plugins/authn/api-server/tokens/plugin.go

```diff
@@ -15,11 +17,13 @@ import (
 
 const PluginName = "tokens"
 
-type plugin struct{}
+type plugin struct {
+	isInitialised bool
```

Comment:
Why is this required?

Reply:
Can you please add a little comment that references this issue for the future reader :)